src/python-frontend/python_converter.cpp

```diff
@@ -1147,6 +1176,26 @@ size_t get_type_size(const nlohmann::json &ast_node)
   return type_size;
 }
 
+std::string python_converter::get_var_type(const std::string &var_name) const
+{
+  nlohmann::json ref;
+  // Get variable from current function
+  for (const auto &elem : ast_json["body"])
+  {
+    if (elem["_type"] == "FunctionDef" && elem["name"] == current_func_name)
+      ref = find_var_decl(var_name, elem);
+  }
+
+  // Get variable from global scope
+  if (ref.empty())
+    ref = find_var_decl(var_name, ast_json);
+
+  if (ref.empty())
```

Comment:
Why do we return `std::string()`?

Reply:
We return an empty string when the variable is not found in the program.